Kickstarter-iOS/DataSources/PledgeDataSourceTests.swift

```diff
@@ -4,15 +4,15 @@ import XCTest
 @testable import KsApi
 
 final class PledgeDataSourceTests: XCTestCase {
-  let collectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewLayout())
```

Comment:
I must have copy-pasted this from a different data source test forgetting about the fact that we're actually dealing with `UITableView`.